cartographer/mapping/internal/3d/pose_graph_3d.cc

```diff
@@ -546,7 +546,7 @@ void PoseGraph3D::WaitForAllComputations() {
 
   // Now wait for any pending constraint computations to finish.
   common::MutexLocker locker(&mutex_);
-  bool notification GUARDED_BY(mutex_) = false;
+  bool notification = false;
```

Comment:
@wohe I don't think this does anything useful here does it?

Reply:
I agree. Let's remove it.